src/homework09/Part2T28Generic.java

```diff
@@ -0,0 +1,51 @@
+package homework09;
+
+//  Написать класс, который умеет хранить в себе массив любых типов данных (int, long etc.).
+//  Реализовать метод get(int index), который возвращает любой элемент массива по индексу.
+
+import java.util.ArrayList;
+
+public class Part2T28Generic {
+    public static void main(String[] args) {
+
+        ElementsClass<Object> elements = new ElementsClass<>();
+        elements.add("String");
+        elements.add('5');
+        elements.add(5);
+        elements.add((char) 55);
+
+        System.out.println(elements.get(0));
+        System.out.println(elements.get(1));
+        System.out.println(elements.get(2));
+        System.out.println(elements.get(3));
+        System.out.println(elements.get(4));
+        System.out.println(elements.toString());
+
+    }
+
+    public static class ElementsClass<T> {
+        private ArrayList<T> elements = new ArrayList<T>();
+
+        public void add(T element) {
+            elements.add(element);
+        }
+
+        public T get(int index) {
```

Comment:
Nice that you tried to handle exception, better to user if and check if elements are not null and elements size is bigger then required index

Reply:
fixed

Reply:
> Nice that you tried to handle exception, better to user if and check if elements are not null and elements size is bigger then required index

fixed